feeds/wifi-trunk/ath10k-ct/patches/970-accumulate-survey-info-data

```diff
@@ -0,0 +1,11 @@
+--- a/ath10k-5.7/mac.c	2021-07-06 11:12:56.022146449 -0700
++++ b/ath10k-5.7/mac.c	2021-07-06 19:37:52.352753693 -0700
+@@ -8286,7 +8286,7 @@
+ 				  struct ieee80211_channel *channel)
+ {
+ 	int ret;
+-	enum wmi_bss_survey_req_type type = WMI_BSS_SURVEY_REQ_TYPE_READ_CLEAR;
++	enum wmi_bss_survey_req_type type = WMI_BSS_SURVEY_REQ_TYPE_READ;
```

Comment:
No need to change anything to calculate deltas and handle wraps?

Reply:
wraps are expected to be handled in firmware. deltas should be taken care in user space